modules/api/src/main/java/org/apache/ignite/table/KeyValueView.java

```diff
@@ -291,9 +317,10 @@ default V getOrDefault(K key, V defaultValue) {
      *
      * @param key A key with which the specified value is associated. The key cannot be {@code null}.
      * @param val Value to be associated with the specified key.
+     * @param tx  The transaction or {@code null} to auto commit.
```

Comment:
The code snippet in the Javadoc for this method should be updated as well.
```
     * if (view.containsKey(key, tx)) {
     *   view.put(key, value, tx);
     *   return true;
     * } else {
     *   return false;
     * }</code></pre>
 ```

Reply:
tx param added.